android/src/main/kotlin/dev/rexios/polar/PolarPlugin.kt

```diff
@@ -124,6 +123,13 @@ class PolarPlugin :
         call: MethodCall,
         result: Result,
     ) {
+        if (wrapperInternal == null) {
```

Comment:
Put all this logic into the `initSdk` method so it can be reused in the `searchHandler`